pkg/export/export.go

```diff
@@ -225,9 +228,20 @@ func (alwaysLease) OnLeaderChange(f func()) {
 	// We never lose the lease as it's always owned.
 }
 
+func formatUserAgent(ua, env string) string {
+	re := regexp.MustCompile(`(.*\()(mode:.*\))$`)
```

Comment:
Where is mode set?

Reply:
It's set in the YAML manifests, for example:
- collectors: https://github.com/GoogleCloudPlatform/prometheus-engine/blob/2e01f7369ed284c77a401c63b53b145eeeec87b1/manifests/operator.yaml#L611
- rule-evaluator: https://github.com/GoogleCloudPlatform/prometheus-engine/blob/2e01f7369ed284c77a401c63b53b145eeeec87b1/manifests/operator.yaml#L779